sources/Application/UI/Base/Component/DataTable/tTableRowActions.php

```diff
@@ -33,11 +31,11 @@ trait tTableRowActions
 	 *      confirmation => {
 	 *          message: string,
 	 *          message_row_data: string,
-	 *          remember_choice_pref_key: string
+	 *          do_not_show_again_pref_key: string
 	 *      }
 	 * }
 	 */
-	protected $aRowActions;
+	protected $aRowActions = array();
```

Comment:
```suggestion
	protected $aRowActions = [];
```

Short array syntax is preferred